packages/react/src/host/Renderer.tsx

```diff
@@ -26,6 +26,7 @@ export const RemoteRenderer = memo(({components, receiver}: Props) => {
           component={child}
           receiver={receiver}
           controller={controller}
+          __type__={(controller.get(child.type) as any)?.__type__}
```

Comment:
nit: can we make this a variable for readability?

Reply:
Can we add a test?

Reply:
There is no existing test for RemoteComponent. I prefer to skip it for now.